app/controllers/admin/csv_controller.rb

```diff
@@ -28,6 +28,17 @@ def regular_schedules; end
 
     def services; end
 
+    def upload_services
+      upload_services_params
```

Comment:
You can use this `upload_services_params` method directly in the `ServiceUploader` call. No need to call it separately.